### ISSUE TITLE
Importing data

### ISSUE THREAD

**Issue by FernandaItaly** [msrosenberg/MetaWin#30]
In my specific meta-analysis situation, since not all studies report variances for each effect sizes, I calculate weights using the formula proposed by Adams et al. (1997) (wi= Ne*Nc/Ne+Nc)
![Cattura](https://github.com/msrosenberg/MetaWin/assets/165384294/9ce5b036-e9f8-4a86-8847-4c1b73cf0c5d)

So, if variance of effects sizes variances is required (as reported below in the pciture), is it correct to use the inverse of the weights?

**Comment by msrosenberg** [comment]
Yes, if you want to use the nonparametric weight estimate from the paper, you can import the "variance" estimate for your studies as the inverse of that weight or (Ne+Nc)/(Ne*Nc).  That is exactly what MW2 calculated and output into the "variance" column in the effect size estimator so it would inverse it (1/Vi) to get the weight when performing an analysis.

**Comment by FernandaItaly** [comment]
Thank you for your response. Additionally, rather than importing the effect size as aresponse ratio (RR=Xe/Xc, where Xe and Xc represent the mean of the experimental and control groups respectively) I have chosen to directly import the individual percentage changes of the response ratio (%change Li= (exp(lnRR)-1)*100) to enhance the presentation with the use of the percentages. Will my choice potentially impact the output results?
